struct_test.go

```diff
@@ -486,6 +487,18 @@ func TestRegisterFunc_structArgs(t *testing.T) {
 			t.Fatalf("FloatAndBool(y: false) = %d, want 0", ret)
 		}
 	}
+	{
+		type FourInt32s struct {
+			f0, f1, f2, f3 int32
+		}
+		var FourInt32sFn func(FourInt32s) int32
+		purego.RegisterLibFunc(&FourInt32sFn, lib, "FourInt32s")
+		result := FourInt32sFn(FourInt32s{100, -50, 25, -75})
```

Comment:
Optional: using `100, -127, 4, -100` sounds more random-ish and might be a better test